app/globals.css

```diff
@@ -0,0 +1,46 @@
+.marker {
+  /* Pin */
+  width: 26.2px;
+  height: 41px;
+  background-image: url("/marker.svg");
+  background-size: cover;
+  border-radius: 0;
+}
+
+.soft-story {
+  /* Pin */
+  width: 11px;
+  height: 11px;
+  background-size: cover;
+
+  /* Ellipse 10 */
+  box-sizing: border-box;
+  position: absolute;
+  /* gray/900 */
+  background-color: #171923;
+  /* white */
+  border: 1px solid #FFFFFF;
+  border-radius: 50%
+}
+
+.key {
+  /* Key */
+  width: 16px;
+  height: 16px;
+  border-radius: 50%
+}
+
+.key-soft-story {
+  /* gray/900 */
+  background-color: #171923;
+}
+
+.key-seismic {
+  /* orange/300 */
+  background-color: #f6ad55;
+}
+
+.key-tsunami {
+  /* pink/400 */
+  background-color: #ed64a6;
+}
```

Comment:
I'm reviewing the design right now, and it seems that the current map style needs a few adjustments. The map should have a shadow and slightly rounded corners. :)

Reply:
You're right! I may hold off on that temporarily b/c the design may change to go edge-to-edge. If not, though, will def add in the rounded corners. I did not notice the shadow, so thanks for pointing that out.

Reply:
I added both the rounded corners and drop shadow. There are some slight whitespace issues that cause scrollbars at breakpoints, but FYI I'm going to wait to see if edge-to-edge design wins out to avoid rework.

Reply:
@mantuok this change was done via this commit https://github.com/sfbrigade/datasci-earthquake/pull/88/commits/5c143fd1301802d96a77d809aaa770803e619391 in `page.tsx`